src/amoc_throttle/amoc_throttle.erl

```diff
@@ -18,26 +16,34 @@
          change_rate_gradually/6,
          stop/1]).
 
--define(DEFAULT_INTERVAL, 60000).%% one minute
+-define(DEFAULT_NO_PROCESSES, 10).
+-define(DEFAULT_INTERVAL, 60000). %% one minute
+
 -type name() :: atom().
+-type rate() :: pos_integer().
+-type interval() :: non_neg_integer().
+%% In milliseconds, defaults to 60000 (one minute) when not given.
+%% An interval of 0 means no delay at all, only the number of simultaneous executions will be
+%% controlled, which corresponds to the number of processes started
```

Comment:
Same as for the previous comment I'd move this above the type it describes.

Reply:
And again, needs to be this way, below, limitations of ex_doc 🥲